src/components/pages/Volunteers/AddContactModal.tsx

```diff
@@ -0,0 +1,410 @@
+                <Form.Control
+                  as="select"
+                  custom
+                  onChange={(e: React.ChangeEvent<HTMLSelectElement>) =>
+                    handleRelationshipChange(setFacilityState, e)
+                  }
+                  defaultValue={facilityState}>
+                  {STATES.map((state) => (
+                    <option value={state}>{state}</option>
+                  ))}
+                </Form.Control>
+              </Form.Group>
+            </Col>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Facility Postal"
+                value={facilityPostal}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                  handleInputChange(setFacilityPostal, e)
+                }
+                isInvalid={errors.facilityPostalErr !== ''}
+              />
+              <Form.Control.Feedback type="invalid">
+                {errors.facilityPostalErr}
+              </Form.Control.Feedback>
+            </Col>
+          </Form.Row>
+
+          <Form.Row className="mb-3">
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Facility Phone"
+                value={facilityPhone}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                  handleInputChange(setFacilityPhone, e)
+                }
+              />
+            </Col>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Dorm"
+                value={facilityDorm}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                  handleInputChange(setFacilityDorm, e)
+                }
+              />
+            </Col>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Unit"
+                value={facilityUnit}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                  handleInputChange(setFacilityUnit, e)
+                }
+              />
+            </Col>
+            <Col>
+              <Form.Group controlId="selectRelationship">
+                <Form.Control
+                  as="select"
+                  custom
+                  onChange={(e: React.ChangeEvent<HTMLSelectElement>) =>
+                    handleRelationshipChange(setRelationship, e)
+                  }
+                  defaultValue={relationship}>
+                  {/* TODO: Get full list of options */}
+                  <option value="Client">Client</option>
+                  <option value="Other">Other</option>
+                </Form.Control>
+              </Form.Group>
+            </Col>
+          </Form.Row>
+        </Form>
+        <div className={vcState.failedUpload ? '' : 'hidden'}>
+          <h4>Oof - we're sorry. Having some trouble with our server.</h4>
```

Comment:
Maybe:

"Oh no. We're having some trouble with our server!"

"We already notified our engineering team, but feel free to email us at support@ameelio.org with more details and we'll take care of this as soon as we can."

And we can also add a cute SVG here!

Reply:
`<svg width="191" height="155" viewBox="0 0 191 155" fill="none" xmlns="http://www.w3.org/2000/svg">
<path opacity="0.1" d="M97.7371 154.002C145.158 154.002 183.6 119.527 183.6 77.0009C183.6 34.4745 145.158 0 97.7371 0C50.3162 0 11.8741 34.4745 11.8741 77.0009C11.8741 119.527 50.3162 154.002 97.7371 154.002Z" fill="#2F80ED"/>
<path opacity="0.1" d="M31.6593 121.32C49.1443 121.32 63.3186 119.112 63.3186 116.389C63.3186 113.666 49.1443 111.458 31.6593 111.458C14.1744 111.458 0 113.666 0 116.389C0 119.112 14.1744 121.32 31.6593 121.32Z" fill="#2F80ED"/>
<path opacity="0.1" d="M46.876 144.811C53.6768 144.811 59.19 143.667 59.19 142.255C59.19 140.844 53.6768 139.699 46.876 139.699C40.0752 139.699 34.562 140.844 34.562 142.255C34.562 143.667 40.0752 144.811 46.876 144.811Z" fill="#2F80ED"/>
<path opacity="0.1" d="M131.018 134.977C137.819 134.977 143.332 133.833 143.332 132.421C143.332 131.01 137.819 129.865 131.018 129.865C124.217 129.865 118.704 131.01 118.704 132.421C118.704 133.833 124.217 134.977 131.018 134.977Z" fill="#2F80ED"/>
<path opacity="0.1" d="M89.2332 132.832C100.292 132.832 109.258 131.306 109.258 129.423C109.258 127.54 100.292 126.014 89.2332 126.014C78.174 126.014 69.2087 127.54 69.2087 129.423C69.2087 131.306 78.174 132.832 89.2332 132.832Z" fill="#2F80ED"/>
<path opacity="0.1" d="M148.235 119.123C171.302 119.123 190.002 116.18 190.002 112.549C190.002 108.919 171.302 105.976 148.235 105.976C125.169 105.976 106.469 108.919 106.469 112.549C106.469 116.18 125.169 119.123 148.235 119.123Z" fill="#2F80ED"/>
<path d="M8.65661 101.077C7.71842 102.911 7.28023 104.962 7.3866 107.021C7.49296 109.08 8.14009 111.074 9.26219 112.801C9.63032 113.356 9.969 113.755 10.2396 113.861C11.5372 114.373 14.1362 115.633 14.1362 115.633L20.3945 113.162C20.3945 113.162 21.4105 107.761 22.3713 103.124C23.0266 99.9614 23.6598 97.1556 23.9193 96.6636C24.5673 95.4521 23.9193 82.2629 23.9193 82.2629C23.9193 82.2629 17.6611 81.8412 15.9401 86.6889L15.1873 88.1686L8.65661 101.077Z" fill="#A8A8A8"/>
<path opacity="0.1" d="M16.9653 102.534C20.77 106.916 22.375 103.126 22.375 103.126C23.0303 99.9632 23.6635 97.1574 23.923 96.6654C24.5709 95.454 23.923 82.2648 23.923 82.2648C23.923 82.2648 17.6648 81.8431 15.9438 86.6908L15.191 88.1704C14.7087 92.0489 14.232 99.3806 16.9653 102.534Z" fill="white"/>
<path d="M17.801 114.747L39.8889 112.183L43.8297 86.1267L40.8497 79.3592C39.5907 76.4771 37.8473 73.8343 35.6958 71.5466C34.1957 69.9579 32.3827 68.4301 30.4757 67.6626C30.2151 67.557 29.9491 67.4656 29.6787 67.3888C29.3679 67.294 29.0392 67.2742 28.7194 67.3311C28.3995 67.3879 28.0976 67.5199 27.8381 67.7162C25.1139 69.6564 24.1402 77.9054 23.8254 81.8764C23.6959 83.5082 23.4766 85.1315 23.1683 86.7389L17.801 114.747Z" fill="#A8A8A8"/>
<path opacity="0.1" d="M32.506 85.8455L27.9044 97.2203C30.3156 95.4484 33.8883 85.7993 33.8883 85.7993L37.0893 96.8005L34.5381 85.2056C34.9116 82.6876 35.0202 80.1372 34.8621 77.5965C34.6596 74.6852 31.9409 70.0132 30.4647 67.6606C30.2041 67.5551 29.9381 67.4637 29.6677 67.3869C29.3569 67.292 29.0282 67.2722 28.7084 67.3291C28.3885 67.386 28.0866 67.5179 27.8271 67.7142C28.9867 70.0539 31.8857 75.9041 32.7692 77.6427C33.8902 79.8326 32.506 85.8455 32.506 85.8455Z" fill="white"/>
<path d="M36.2628 115.633H48.2473C48.7403 115.633 49.2281 115.532 49.6812 115.337C50.1343 115.142 50.5431 114.856 50.8828 114.497C51.2225 114.138 51.486 113.713 51.6572 113.249C51.8285 112.784 51.9039 112.29 51.8789 111.795C51.6139 106.631 50.942 97.4237 49.3443 91.5791C48.4811 88.4256 47.4779 86.7702 46.5576 85.9546C46.126 85.5361 45.5688 85.2727 44.9728 85.2055C44.5207 85.1663 44.0682 85.2767 43.6843 85.5199C41.459 87.6636 38.86 98.944 38.86 98.944L36.2628 115.633Z" fill="#A8A8A8"/>
<path opacity="0.1" d="M47.327 105.628C47.8203 107.003 48.0154 108.588 47.327 110.132L44.6562 111.458C44.6562 111.458 37.4776 108.401 38.3501 102.135C38.3501 102.135 38.5857 100.84 41.0577 100.94C42.4479 101.007 43.7867 101.489 44.9032 102.324C46.0198 103.159 46.8635 104.31 47.327 105.628Z" fill="white"/>
<path opacity="0.1" d="M44.474 89.1562C44.3911 86.9589 45.5784 86.1969 46.5594 85.9546C46.1278 85.5361 45.5707 85.2728 44.9746 85.2056C44.2917 85.7798 43.8077 86.5569 43.5923 87.425L41.089 99.6858C41.089 99.6858 44.6138 92.8387 44.474 89.1562Z" fill="white"/>
<path d="M33.1098 98.8517L39.044 107.008L40.073 113.741L36.2591 115.633L30.7611 116.853C30.2316 116.972 29.6871 117.007 29.1468 116.959L14.1436 115.633L16.1849 110.878L23.2346 100.932L33.1098 98.8517Z" fill="#A8A8A8"/>
<path opacity="0.1" d="M30.8126 96.0514C30.8126 96.0514 34.8621 96.3307 33.3417 103.322C31.8213 110.313 16.1776 110.874 16.1776 110.874C16.1776 110.874 19.5018 104.955 24.6151 100.714C25.1894 100.239 25.7434 99.7432 26.2717 99.2142C27.4479 98.0323 29.6972 95.9553 30.8126 96.0514Z" fill="white"/>
<path opacity="0.1" d="M34.4553 101.323C34.4553 101.323 39.9423 106.34 39.425 114.062L34.4553 101.323Z" fill="black"/>
<path opacity="0.1" d="M43.6861 85.52C43.6861 85.52 37.9286 93.2142 37.2825 105.053L43.6861 85.52Z" fill="black"/>
<path opacity="0.1" d="M23.9249 82.2592C23.9249 82.2592 26.7116 94.9749 19.7539 104.559L23.9249 82.2592Z" fill="black"/>
<path d="M74.681 129.353H104.71C105 129.351 105.285 129.274 105.537 129.129C105.788 128.984 105.999 128.776 106.147 128.526C106.295 128.275 106.376 127.99 106.382 127.698C106.389 127.407 106.32 127.118 106.182 126.861L91.1751 98.8443C91.0313 98.5757 90.8178 98.3513 90.5572 98.1949C90.2967 98.0384 89.9988 97.9558 89.6953 97.9558C89.3917 97.9558 89.0939 98.0384 88.8333 98.1949C88.5727 98.3513 88.3592 98.5757 88.2154 98.8443L73.2011 126.861C73.0633 127.119 72.9943 127.408 73.0008 127.7C73.0073 127.992 73.089 128.278 73.2381 128.529C73.3872 128.78 73.5985 128.988 73.8514 129.132C74.1043 129.276 74.3901 129.353 74.681 129.353ZM91.6059 124.233H87.7847V120.395H91.6059V124.233ZM91.6059 117.836H87.7847V110.158H91.6059V117.836Z" fill="#464353"/>
<path opacity="0.1" d="M74.681 129.353H104.71C105 129.351 105.285 129.274 105.537 129.129C105.788 128.984 105.999 128.776 106.147 128.526C106.295 128.275 106.376 127.99 106.382 127.698C106.389 127.407 106.32 127.118 106.182 126.861L91.1751 98.8443C91.0313 98.5757 90.8178 98.3513 90.5572 98.1949C90.2967 98.0384 89.9988 97.9558 89.6953 97.9558C89.3917 97.9558 89.0939 98.0384 88.8333 98.1949C88.5727 98.3513 88.3592 98.5757 88.2154 98.8443L73.2011 126.861C73.0633 127.119 72.9943 127.408 73.0008 127.7C73.0073 127.992 73.089 128.278 73.2381 128.529C73.3872 128.78 73.5985 128.988 73.8514 129.132C74.1043 129.276 74.3901 129.353 74.681 129.353ZM91.6059 124.233H87.7847V120.395H91.6059V124.233ZM91.6059 117.836H87.7847V110.158H91.6059V117.836Z" fill="black"/>
<path d="M74.681 130.277H104.71C105 130.276 105.285 130.199 105.537 130.054C105.788 129.909 105.999 129.701 106.147 129.45C106.295 129.199 106.376 128.914 106.382 128.623C106.389 128.331 106.32 128.043 106.182 127.786L91.1751 99.769C91.0313 99.5004 90.8178 99.276 90.5572 99.1195C90.2967 98.9631 89.9988 98.8805 89.6953 98.8805C89.3917 98.8805 89.0939 98.9631 88.8333 99.1195C88.5727 99.276 88.3592 99.5004 88.2154 99.769L73.2011 127.786C73.0633 128.044 72.9943 128.332 73.0008 128.625C73.0073 128.917 73.089 129.202 73.2381 129.453C73.3872 129.704 73.5985 129.912 73.8514 130.057C74.1043 130.201 74.3901 130.277 74.681 130.277ZM91.6059 125.158H87.7847V121.32H91.6059V125.158ZM91.6059 118.76H87.7847V111.083H91.6059V118.76Z" fill="#464353"/>
<path d="M43.0364 43.7829C43.0161 44.1694 43.2076 44.5227 43.3198 44.8926C43.5056 45.607 43.4871 46.3596 43.2665 47.064C43.0603 47.8038 42.7143 48.5862 42.9775 49.3093C43.0382 49.4795 43.1303 49.6349 43.1855 49.805C43.4137 50.5079 42.9904 51.2477 42.5265 51.8229L44.3469 53.1046C44.6127 53.326 44.9289 53.4779 45.2673 53.5467C45.6078 53.5874 46.0035 53.4043 46.0735 53.0677C46.0974 52.953 46.0735 52.8328 46.0956 52.7162C46.118 52.5656 46.1759 52.4225 46.2646 52.2989C46.3533 52.1753 46.4702 52.0749 46.6054 52.006C46.8788 51.8713 47.1838 51.8143 47.4871 51.8414C48.6044 51.9024 49.5542 52.6404 50.5702 53.1139C51.2401 53.4054 51.9342 53.6368 52.6446 53.8056C53.0477 53.9111 53.495 54.0072 53.8595 53.8056C54.2239 53.604 54.3914 53.1361 54.3638 52.7125C54.3072 52.2913 54.1763 51.8836 53.9773 51.5085C53.7764 51.1338 53.6554 50.7212 53.622 50.297C53.622 49.6756 53.9607 49.1096 54.0748 48.4992C54.235 47.6318 53.9092 46.7496 53.5116 45.9616C53.114 45.1737 52.6317 44.4154 52.4385 43.5572C52.4001 43.2971 52.3328 43.0421 52.2378 42.797C52.1457 42.6109 52.021 42.443 51.8697 42.3014C51.1868 41.6226 50.2131 41.3007 49.2596 41.2305C48.3062 41.1602 47.3435 41.3174 46.3974 41.482C46.0365 41.5338 45.681 41.6186 45.3354 41.7354C44.8415 41.9326 44.3726 42.1878 43.9383 42.4956C43.5076 42.7952 43.064 43.2335 43.0364 43.7829Z" fill="#464353"/>
<g opacity="0.1">
<path opacity="0.1" d="M53.6165 49.9179C53.6165 49.9733 53.6165 50.0288 53.6276 50.0825C53.6846 49.5405 53.9681 49.0319 54.0693 48.49C54.1084 48.2728 54.1202 48.0516 54.1043 47.8315C54.1043 47.9277 54.0859 48.0165 54.0693 48.1201C53.9534 48.7304 53.6091 49.2964 53.6165 49.9179Z" fill="black"/>
<path opacity="0.1" d="M43.4432 45.6324C43.4649 45.2581 43.4232 44.8829 43.3199 44.5227C43.2109 44.242 43.1218 43.954 43.053 43.6608C43.053 43.6996 43.0383 43.7366 43.0364 43.7754C43.0162 44.162 43.2076 44.5153 43.3199 44.8852C43.391 45.1283 43.4324 45.3792 43.4432 45.6324Z" fill="black"/>
<path opacity="0.1" d="M54.3473 52.6108C54.3344 52.7761 54.283 52.936 54.1974 53.0777C54.1117 53.2194 53.994 53.3389 53.854 53.4265C53.4858 53.6281 53.0422 53.5319 52.6391 53.4265C51.9285 53.2579 51.2342 53.0258 50.5647 52.7329C49.5487 52.2613 48.5989 51.5233 47.4816 51.4622C47.1783 51.4352 46.8733 51.4921 46.5999 51.6269C46.4647 51.6958 46.3478 51.7962 46.2591 51.9197C46.1704 52.0433 46.1125 52.1864 46.0901 52.3371C46.0772 52.4536 46.0901 52.5738 46.068 52.6885C45.9962 53.0251 45.6023 53.2082 45.2618 53.1675C44.9234 53.0987 44.6072 52.9469 44.3414 52.7255L42.7051 51.5714C42.6444 51.6546 42.5836 51.736 42.521 51.8118L44.3414 53.0954C44.6072 53.3168 44.9234 53.4686 45.2618 53.5374C45.6023 53.5781 45.998 53.395 46.068 53.0584C46.0919 52.9437 46.068 52.8235 46.0901 52.707C46.1125 52.5563 46.1704 52.4132 46.2591 52.2896C46.3478 52.1661 46.4647 52.0657 46.5999 51.9968C46.8733 51.862 47.1783 51.8051 47.4816 51.8322C48.5989 51.8932 49.5487 52.6312 50.5647 53.1028C51.2342 53.3957 51.9285 53.6278 52.6391 53.7964C53.0422 53.9018 53.4895 53.998 53.854 53.7964C54.2184 53.5948 54.3859 53.1268 54.3583 52.7033C54.3565 52.6719 54.3509 52.6423 54.3473 52.6108Z" fill="black"/>
<path opacity="0.1" d="M43.1855 49.7958C43.2082 49.8702 43.2248 49.9462 43.2352 50.0233C43.2637 49.823 43.2467 49.6187 43.1855 49.4259C43.1303 49.2557 43.0382 49.1004 42.9775 48.9302C42.9332 48.8067 42.9048 48.6781 42.8928 48.5474C42.8656 48.8015 42.8945 49.0585 42.9775 49.3001C43.0382 49.4703 43.1303 49.6275 43.1855 49.7958Z" fill="black"/>
</g>
<path d="M45.3464 119.977C45.3338 120.03 45.304 120.077 45.2618 120.111C45.217 120.134 45.1668 120.145 45.1164 120.142C44.4611 120.157 43.8058 120.142 43.1524 120.188C42.5339 120.242 41.9136 120.373 41.297 120.303C41.0997 120.31 40.9052 120.255 40.7403 120.146C40.5754 120.037 40.4483 119.879 40.3767 119.694C40.3378 119.614 40.3034 119.531 40.2736 119.447C40.1037 119.011 40.0181 118.546 40.0214 118.078C40.0214 117.954 40.0214 117.834 40.0214 117.708C40.0134 117.572 39.9955 117.438 39.968 117.305C39.9122 117.152 39.9188 116.984 39.9864 116.837C40.2478 116.358 40.8792 116.262 41.3651 116.225C42.4909 116.139 43.623 116.223 44.7243 116.472C44.8108 116.898 44.9084 117.32 45.0114 117.741C45.0875 118.051 45.1685 118.359 45.2544 118.666C45.312 118.856 45.3588 119.049 45.3943 119.245C45.4398 119.489 45.4234 119.741 45.3464 119.977Z" fill="#464353"/>
<path opacity="0.06" d="M45.3464 119.977C45.3338 120.03 45.304 120.077 45.2617 120.111C45.2169 120.134 45.1667 120.145 45.1163 120.142C44.461 120.157 43.8058 120.142 43.1523 120.188C42.5339 120.242 41.9136 120.373 41.297 120.303C41.0997 120.31 40.9051 120.255 40.7403 120.146C40.5754 120.037 40.4483 119.879 40.3766 119.694C40.3377 119.614 40.3033 119.531 40.2736 119.447H40.5092C41.2841 119.447 42.0608 119.419 42.8358 119.419C43.0695 119.419 43.3051 119.408 43.537 119.38C43.769 119.352 43.9586 119.313 44.1702 119.287C44.4871 119.257 44.8054 119.244 45.1237 119.249H45.3851C45.4329 119.491 45.4196 119.741 45.3464 119.977Z" fill="black"/>
<path d="M58.9029 116.543C58.6977 116.783 58.4357 116.968 58.1409 117.081C56.8303 117.636 55.3154 117.386 53.9626 116.935C53.6599 116.822 53.3492 116.732 53.033 116.667C52.7446 116.626 52.4537 116.606 52.1624 116.606C51.9544 116.606 51.7464 116.589 51.5384 116.576C51.012 116.543 50.4874 116.491 49.9647 116.439C49.6002 116.412 49.2384 116.357 48.8823 116.275C48.7811 116.248 48.6839 116.209 48.5934 116.156C48.5161 116.122 48.4502 116.067 48.4032 115.996C48.3563 115.926 48.3303 115.843 48.3283 115.759C48.343 115.524 48.586 115.389 48.7995 115.287C50.6936 114.464 52.7514 114.107 54.7945 113.82C55.0024 113.785 55.2047 113.723 55.3964 113.635C55.6265 113.543 55.6504 113.694 55.7646 113.865C55.8185 113.943 55.8802 114.016 55.9486 114.083C56.1705 114.284 56.4311 114.438 56.7143 114.534C57.2113 114.719 57.7433 114.769 58.2403 114.945C58.3906 114.997 58.5345 115.067 58.6691 115.152C58.788 115.217 58.8904 115.308 58.9687 115.419C59.0469 115.53 59.0989 115.658 59.1205 115.792C59.1422 115.926 59.133 116.064 59.0937 116.194C59.0544 116.324 58.986 116.443 58.8937 116.543H58.9029Z" fill="#464353"/>
<path opacity="0.06" d="M58.9029 116.543C58.6977 116.783 58.4357 116.968 58.1408 117.081C56.8303 117.636 55.3154 117.386 53.9625 116.935C53.6599 116.822 53.3492 116.732 53.033 116.667C52.7446 116.626 52.4536 116.606 52.1624 116.606C51.9544 116.606 51.7464 116.589 51.5384 116.576C51.7077 116.415 51.8808 116.258 52.0575 116.106C52.1855 115.986 52.3331 115.888 52.4937 115.818C52.6937 115.752 52.9069 115.737 53.114 115.775C54.1668 115.921 55.1129 116.663 56.175 116.598C56.6698 116.541 57.1484 116.386 57.5831 116.141C57.9366 115.997 58.2517 115.771 58.5034 115.483C58.5794 115.384 58.6373 115.271 58.6746 115.152C58.7935 115.217 58.8959 115.308 58.9742 115.419C59.0524 115.53 59.1044 115.658 59.126 115.792C59.1477 115.926 59.1385 116.064 59.0992 116.194C59.0599 116.324 58.9915 116.443 58.8992 116.543H58.9029Z" fill="black"/>
<path d="M58.5329 84.3991C58.3489 84.769 58.1133 85.1704 57.7157 85.285C57.1377 85.4515 56.5818 84.9299 56.3297 84.3825C55.9859 83.5789 55.9211 82.6825 56.1456 81.8375C56.2505 81.4102 56.4106 80.9996 56.5008 80.5705C56.5659 80.2094 56.6053 79.8441 56.6186 79.4774C56.673 78.9304 56.66 78.3788 56.58 77.835C56.2836 76.2462 54.9234 75.0607 54.4706 73.4996C54.2423 72.721 54.2534 71.8924 54.2663 71.0804C54.262 70.8752 54.283 70.6701 54.3289 70.4701C54.3425 70.4155 54.361 70.3623 54.3841 70.311C54.6878 69.6341 55.5879 69.5416 56.2137 69.1476C56.5597 68.9312 57.3457 68.234 57.6365 68.8702C57.6993 69.0119 57.7462 69.1601 57.7764 69.3122C57.8961 69.8782 57.8482 70.5422 57.8777 71.0767C57.9899 73.0964 58.4906 75.1125 58.2587 77.1211C58.2034 77.4635 58.1763 77.8099 58.1777 78.1568C58.2274 79.0261 58.7685 79.7789 59.017 80.6112C59.3999 81.8634 59.0925 83.2265 58.5329 84.3991Z" fill="#FBBEBE"/>
<path opacity="0.1" d="M57.7856 69.316C57.6489 69.2661 57.499 69.2661 57.3623 69.316L56.1161 69.6859C55.6795 69.8029 55.2539 69.9583 54.8443 70.1501C54.6654 70.2383 54.4959 70.3443 54.3381 70.4664C54.3518 70.4119 54.3703 70.3586 54.3933 70.3074C54.697 69.6304 55.5971 69.5379 56.2229 69.144C56.5689 68.9276 57.3549 68.2303 57.6457 68.8665C57.709 69.0106 57.756 69.1614 57.7856 69.316Z" fill="black"/>
<path d="M57.8666 69.1755C57.7915 69.1296 57.7068 69.1021 57.6192 69.095C57.5317 69.0879 57.4437 69.1016 57.3623 69.1348L56.1161 69.5047C55.6795 69.6217 55.2539 69.777 54.8442 69.9689C54.6269 70.0752 54.4235 70.2081 54.2387 70.3647C53.657 69.3493 53.5024 68.1453 53.3956 66.98C53.3367 66.3012 53.2926 65.6206 53.2668 64.9455C53.2379 64.4597 53.2478 63.9723 53.2963 63.4881C53.3662 62.9332 53.5263 62.3913 53.6257 61.8401C53.6849 61.5174 53.7231 61.1912 53.7399 60.8635C53.7876 60.1476 53.7549 59.4285 53.6423 58.7199C53.5353 57.9667 53.3641 57.2241 53.1306 56.5004C53.0128 56.1693 52.8711 55.8494 52.7422 55.5238L52.698 55.4092C52.4653 54.8126 52.2753 54.2 52.1293 53.5763C52.0704 53.321 52.0667 53.0528 51.8716 52.879C52.0548 52.8568 52.2394 52.8494 52.4238 52.8568C52.4606 52.8568 52.4974 52.8568 52.5342 52.8568C53.2904 52.9046 54.0189 53.1605 54.6399 53.5966C55.4398 54.1765 56.0477 54.9846 56.3849 55.9159C56.7733 56.9573 56.8377 58.0892 56.8874 59.2008C56.9868 61.3857 57.0481 63.5719 57.0714 65.7593C57.0714 66.1792 57.0899 66.6268 57.3291 66.9726C57.3963 67.0579 57.4578 67.1475 57.5132 67.2408C57.5997 67.4258 57.5868 67.6311 57.6568 67.8179C57.7081 67.93 57.7666 68.0387 57.8316 68.1434C57.914 68.3018 57.96 68.4768 57.966 68.6554C57.9721 68.8341 57.9381 69.0118 57.8666 69.1755Z" fill="#464353"/>
<path d="M45.6152 47.4652C45.4311 47.4042 45.247 47.3395 45.063 47.2803L45.4458 47.3099C45.5097 47.3529 45.5667 47.4053 45.6152 47.4652Z" fill="#FBBEBE"/>
<path d="M53.1527 54.1404C53.1158 54.2661 53.079 54.3919 53.0385 54.5103C52.9451 54.8142 52.8313 55.1114 52.698 55.3999C52.4894 55.8739 52.1941 56.3043 51.8274 56.6687C51.7063 56.7968 51.5577 56.8955 51.393 56.9572C51.2536 56.9931 51.1095 57.0068 50.966 56.9979C50.4751 56.9979 49.9843 56.9905 49.4934 56.9757C49.1113 56.98 48.7298 56.9422 48.3559 56.8629C47.9443 56.7501 47.553 56.5728 47.1963 56.3376C45.8213 55.4831 44.6911 54.181 44.2071 52.6385C44.1463 52.4425 44.104 52.2076 44.2383 52.0541C44.3706 51.9457 44.5371 51.8886 44.7077 51.8931H44.7298C45.1255 51.8414 45.5068 51.7099 45.8508 51.5066C46.0736 51.392 46.2532 51.2076 46.3625 50.9813C46.437 50.7131 46.4313 50.4287 46.3459 50.1638L45.9373 48.2995C45.8947 48.0009 45.7831 47.7164 45.6115 47.469C45.7311 47.5115 45.8508 47.5504 45.9796 47.5837C47.1171 47.8925 48.3614 47.7057 49.4769 48.1108C49.9039 48.268 50.3309 48.551 50.4598 48.9912C50.6751 49.731 49.9554 50.4986 50.1616 51.2384C50.2996 51.7415 50.8095 52.0319 51.266 52.2668L51.5734 52.424L52.4256 52.8586L52.5434 52.9178L52.5618 52.927C52.8305 53.0639 53.114 53.2341 53.2079 53.5245C53.2479 53.7309 53.2288 53.9445 53.1527 54.1404Z" fill="#FBBEBE"/>
<path opacity="0.1" d="M55.3983 58.4832C55.3671 58.956 55.2524 59.4194 55.0596 59.8518C54.69 60.5997 54.4919 61.4213 54.4798 62.2563C54.338 64.7532 53.3938 67.7254 54.7467 69.8228C54.7798 69.8746 54.8129 69.9264 54.8442 69.98C54.6268 70.0863 54.4234 70.2192 54.2386 70.3758C53.657 69.3604 53.5024 68.1564 53.3956 66.9911C53.3367 66.3123 53.2925 65.6317 53.2668 64.9566C53.2379 64.4708 53.2477 63.9834 53.2962 63.4992C53.3662 62.9443 53.5263 62.4024 53.6257 61.8512C53.6849 61.5285 53.723 61.2023 53.7398 60.8746C53.7876 60.1587 53.7548 59.4396 53.6423 58.731C53.5352 57.9778 53.364 57.2353 53.1306 56.5115C53.0128 56.1804 52.871 55.8605 52.7422 55.535L52.698 55.4203C52.4894 55.8942 52.1941 56.3246 51.8274 56.6891C51.7063 56.8172 51.5577 56.9159 51.393 56.9776C51.2536 57.0134 51.1095 57.0272 50.9659 57.0183C50.4751 57.0183 49.9843 57.0109 49.4934 56.9961C49.1113 57.0004 48.7298 56.9625 48.3559 56.8833C47.9443 56.7705 47.553 56.5932 47.1963 56.358C45.8213 55.5035 44.6911 54.2014 44.2071 52.6589C44.1463 52.4628 44.104 52.2279 44.2383 52.0744C44.3706 51.966 44.5371 51.9089 44.7077 51.9135H44.7298L46.3864 53.9721C46.5764 54.19 46.7373 54.4318 46.865 54.6916C46.8897 54.7787 46.9412 54.8557 47.0122 54.9116C47.0698 54.9381 47.1331 54.9495 47.1963 54.9449C48.7516 54.9449 50.8242 54.9449 52.3796 54.9449C52.3847 54.668 52.305 54.3962 52.1513 54.1663C52.1035 54.1774 52.0391 54.1811 52.1384 54.1515C52.0485 53.5511 51.8554 52.9709 51.5678 52.4369C51.5507 52.4002 51.5316 52.3644 51.5108 52.3297C51.3872 52.1467 51.2951 51.9441 51.2384 51.7304C51.4347 51.6675 51.6457 51.6675 51.8421 51.7304C52.0685 51.8284 52.1863 52.0744 52.2857 52.3019C52.3704 52.4869 52.455 52.6848 52.5305 52.879C52.5305 52.8993 52.5489 52.9215 52.5563 52.9419C52.7168 53.3246 52.8198 53.7291 52.8618 54.1422C52.8729 54.2939 52.9097 54.4863 53.0348 54.5362C53.0545 54.5452 53.0758 54.5503 53.0974 54.551C53.227 54.5033 53.3678 54.4954 53.5018 54.5284C53.6359 54.5613 53.7572 54.6336 53.8503 54.7359C54.0313 54.9443 54.1578 55.1948 54.2184 55.4647C54.2782 55.7345 54.3658 55.9973 54.4798 56.2489C54.7172 56.7039 54.3546 56.9073 54.7835 57.1866C54.984 57.2975 55.1566 57.4531 55.2878 57.6416C55.4091 57.9042 55.4477 58.1979 55.3983 58.4832Z" fill="black"/>
<path opacity="0.1" d="M44.7077 116.48C44.7942 116.905 44.8918 117.327 44.9949 117.749L44.9838 117.775C44.1426 117.405 43.2812 117.018 42.3627 116.937C41.5105 116.863 40.5847 117.094 40.0049 117.715C39.9968 117.58 39.9789 117.445 39.9515 117.312C39.8956 117.16 39.9023 116.992 39.9699 116.844C40.2313 116.365 40.8626 116.269 41.3485 116.232C42.4743 116.147 43.6064 116.23 44.7077 116.48Z" fill="black"/>
<path opacity="0.1" d="M51.9581 54.5048C52.0446 54.5048 52.0133 54.5048 51.971 54.5196C51.9286 54.5344 51.8587 54.5343 51.9581 54.5048Z" fill="black"/>
<path opacity="0.1" d="M55.9578 114.083C55.137 114.608 54.3656 115.208 53.6533 115.875C53.4465 116.097 53.1999 116.279 52.9263 116.41C52.6437 116.501 52.3453 116.532 52.0501 116.5C50.8922 116.453 49.738 116.338 48.5934 116.156C48.5161 116.122 48.4502 116.067 48.4032 115.996C48.3563 115.926 48.3303 115.843 48.3283 115.759C48.343 115.524 48.586 115.389 48.7995 115.287C50.6936 114.464 52.7514 114.107 54.7945 113.82C55.0024 113.785 55.2047 113.723 55.3964 113.635C55.6265 113.543 55.6504 113.694 55.7646 113.865C55.8214 113.944 55.8861 114.017 55.9578 114.083Z" fill="black"/>
<path d="M42.4529 52.3593C42.7235 52.6016 43.0327 52.7976 43.307 53.0325C44.0432 53.6595 44.5071 54.5362 44.9636 55.3907C44.6929 55.489 44.4001 55.5089 44.1187 55.4481C43.974 56.2686 43.9984 57.1103 44.1905 57.9209C43.8222 58.1748 43.5361 58.5317 43.3677 58.9474C43.2369 59.3693 43.1469 59.8029 43.099 60.2421C42.8339 62.1669 42.5677 64.0917 42.3001 66.0164C42.2948 66.2048 42.2304 66.3867 42.1161 66.5362C42.065 66.5776 42.0181 66.6241 41.9762 66.6749C41.9199 66.7993 41.8976 66.9366 41.9118 67.0725C41.9007 67.2076 41.8124 67.3685 41.678 67.35C41.6127 67.3343 41.5548 67.2964 41.5142 67.2427C41.3798 67.0997 41.2567 66.9463 41.1461 66.784C40.0601 68.8777 39.5373 71.1342 38.963 73.4239C38.5526 75.0552 38.1421 76.7051 38.1219 78.3882C38.1274 79.1844 38.1772 79.9797 38.2709 80.7704L38.4329 82.3943C38.5636 83.7001 38.6943 85.0077 38.836 86.3135C38.9281 87.1421 39.0201 87.9781 39.3164 88.7586C39.7582 89.9442 39.2686 91.2814 39.4563 92.5354C39.6036 93.5176 39.8962 94.533 39.6017 95.4799C39.4839 95.8591 39.2778 96.205 39.1232 96.5712C38.744 97.4812 38.7053 98.4929 38.6722 99.4787C38.6126 100.343 38.6447 101.211 38.7679 102.068C38.9262 102.954 39.2815 103.821 39.2115 104.717C39.114 105.991 38.1789 107.08 38.0648 108.353C37.9746 109.368 38.4182 110.349 38.8949 111.249C39.2465 111.913 38.7569 112.771 38.571 113.502C38.2028 114.934 38.8158 116.432 39.5668 117.702C39.5859 117.746 39.6198 117.781 39.6625 117.802C39.749 117.834 39.8263 117.749 39.8834 117.677C40.4484 116.948 41.4498 116.672 42.3682 116.752C43.2867 116.831 44.1463 117.212 44.9893 117.59L46.1839 114.917C46.3864 114.464 46.5926 113.974 46.4913 113.487C46.4195 113.14 46.1968 112.845 46.0385 112.527C45.6704 111.776 45.652 110.902 45.6483 110.06L45.6262 105.307C46.5576 104.13 47.1263 102.732 47.4853 101.271C48.0359 103.326 48.0739 105.486 47.5957 107.559C47.5072 107.873 47.4457 108.193 47.4116 108.517C47.3399 109.836 48.3688 110.952 48.6431 112.243C48.6696 112.341 48.6771 112.444 48.6651 112.546C48.6343 112.671 48.5826 112.79 48.5124 112.897C48.0191 113.796 48.0724 114.886 48.1442 115.91C49.435 116.136 50.7392 116.276 52.0483 116.33C52.3434 116.36 52.6415 116.329 52.9244 116.239C53.1981 116.108 53.4447 115.926 53.6515 115.703C54.4872 114.913 55.4048 114.216 56.3885 113.622C55.8392 112.706 55.0703 111.943 54.1521 111.403C54.4485 110.964 55.124 111 55.5179 110.648C55.7157 110.452 55.8448 110.196 55.886 109.919C56.0333 109.194 55.8511 108.44 55.6688 107.728C55.3154 106.326 54.9602 104.918 54.3804 103.594C54.1182 103.05 53.8871 102.491 53.6883 101.92C53.206 100.326 53.5429 98.5909 53.1784 96.9614C53.0606 96.4399 52.871 95.9016 53.0036 95.3819C53.206 94.5811 54.108 94.1057 54.3068 93.303C55.0191 90.4454 55.1461 87.4843 55.551 84.5675C55.6416 84.0104 55.6902 83.4471 55.6965 82.8826C55.6965 82.2926 55.5915 81.7081 55.5492 81.12C55.3835 78.7655 56.1327 76.4165 55.9413 74.0639C55.8187 72.6141 55.3433 71.2167 54.5571 69.9948C53.2042 67.8956 54.1485 64.9252 54.2902 62.4283C54.3008 61.5934 54.4984 60.7716 54.8682 60.0239C55.0626 59.5919 55.1779 59.1283 55.2087 58.6552C55.2654 58.3635 55.2294 58.0613 55.1056 57.7915C54.9761 57.6022 54.8039 57.4463 54.6031 57.3365C54.1742 57.0553 54.5368 56.8519 54.2994 56.3969C54.1852 56.1454 54.0976 55.8825 54.038 55.6127C53.9778 55.3432 53.8512 55.0933 53.6699 54.8858C53.5768 54.7834 53.4555 54.7112 53.3215 54.6782C53.1874 54.6453 53.0466 54.6532 52.9171 54.7008C52.7422 54.7008 52.6943 54.4678 52.6796 54.2921C52.6299 53.6484 52.3649 53.0436 52.1053 52.4518C52.0059 52.2243 51.8881 51.9783 51.6617 51.8803C51.4646 51.8183 51.2533 51.8183 51.0562 51.8803C51.1131 52.0947 51.2059 52.298 51.3304 52.4814C51.6495 53.0438 51.8617 53.6609 51.9562 54.3013C51.7722 54.3568 52.1403 54.3013 51.9562 54.3013C52.1211 54.5321 52.2058 54.8109 52.1974 55.0948C50.6438 55.0948 48.5713 55.0948 47.0159 55.0948C46.9527 55.0998 46.8893 55.0883 46.8318 55.0615C46.7609 55.0052 46.7089 54.9284 46.6828 54.8414C46.5562 54.5816 46.3952 54.3402 46.2042 54.1238L44.3819 51.8655C44.3415 51.8064 44.2879 51.7577 44.2255 51.723C44.1502 51.7017 44.0706 51.7017 43.9954 51.723C43.4045 51.81 42.9683 52.0578 42.4529 52.3593Z" fill="#2F80ED"/>
<path opacity="0.1" d="M48.9928 82.9583C49.3069 82.954 49.6165 83.0345 49.8892 83.1913C50.0245 83.2714 50.1367 83.3854 50.2149 83.5223C50.293 83.6591 50.3342 83.8141 50.3346 83.9719C50.3144 84.3214 50.0475 84.5989 49.8542 84.8966C49.7346 85.0781 49.6393 85.2746 49.5707 85.4811C49.5496 85.5281 49.5444 85.5807 49.556 85.6309C49.5781 85.7012 49.6536 85.7363 49.718 85.7678C49.9522 85.8742 50.1522 86.0441 50.2956 86.2584C50.439 86.4728 50.52 86.723 50.5297 86.9811C50.5132 87.7024 49.8119 88.2869 49.8505 89.0156C49.8681 89.1674 49.8939 89.318 49.9278 89.4669C50.012 90.088 49.9261 90.7204 49.6793 91.2961C49.4566 91.8769 49.1529 92.4225 48.9136 92.9977C48.2676 94.5606 48.1277 96.2918 48.1774 97.9915C48.216 99.1457 47.8092 100.244 47.4411 101.343C47.6491 99.13 47.7045 96.9052 47.6068 94.6845C47.581 94.0816 47.5055 93.5119 47.5331 92.9071C47.5773 91.9472 48.1111 91.0742 48.2473 90.1328C48.3246 89.5945 48.2731 89.0471 48.286 88.5033C48.3117 87.3935 48.6118 86.2986 48.632 85.1852C48.6578 84.3991 48.586 83.589 48.9928 82.9583Z" fill="black"/>
<path d="M41.3118 52.7551C40.9458 52.9384 40.6034 53.1657 40.292 53.432C39.9667 53.7568 39.6946 54.1314 39.4858 54.5417C39.2316 55.0069 39.0036 55.4861 38.8029 55.977C38.5434 56.6132 38.328 57.3179 38.5066 57.9819C38.6686 58.583 39.1269 59.051 39.4435 59.5855C39.7601 60.12 39.946 60.747 40.1889 61.3333C40.5129 62.1157 40.9565 62.8444 41.2768 63.6286C41.5971 64.4128 41.7903 65.2932 41.5878 66.1089C41.5289 66.3493 41.1406 66.5361 41.1406 66.7821C41.1406 67.189 41.7259 67.6311 42.1143 67.7494C42.5616 67.8882 43.0493 67.681 43.4156 67.3795C44.0875 66.845 44.4961 66.0441 44.798 65.2377C45.2839 63.943 45.5434 62.5688 45.7772 61.2038C45.9465 60.2014 46.1048 59.1971 46.195 58.1872C46.2778 57.2772 46.3036 56.3377 46.0643 55.4647C45.841 54.6634 45.4067 53.9375 44.8072 53.3636C44.3286 52.9012 43.5445 52.2797 42.8487 52.2631C42.2983 52.2612 41.783 52.4943 41.3118 52.7551Z" fill="#464353"/>
<path d="M46.5447 75.0515C46.5247 75.1779 46.4777 75.2985 46.4068 75.4048C46.3359 75.5111 46.2429 75.6007 46.1342 75.6674C45.917 75.8001 45.6803 75.8974 45.4329 75.956C44.9396 76.1002 44.4169 76.2334 43.9181 76.1058C43.526 75.9775 43.1641 75.7705 42.8542 75.4973L37.2807 71.21C36.8329 70.8911 36.4212 70.5239 36.0529 70.115C35.8415 69.8511 35.6472 69.5737 35.4713 69.2846C35.2384 68.9536 35.056 68.5896 34.9301 68.2045C34.8852 68.0349 34.8629 67.8602 34.8639 67.6847C34.8624 67.5188 34.9029 67.3552 34.9817 67.2094C35.116 66.9837 35.4032 66.9375 35.6388 66.8746C36.4296 66.6692 37.2504 66.6066 38.0629 66.6897C38.1343 66.6862 38.2049 66.7062 38.264 66.7466C38.3231 66.7869 38.3675 66.8455 38.3906 66.9135C38.5139 67.141 38.6206 67.3777 38.7311 67.6144C38.9006 68.0181 39.1215 68.398 39.3882 68.7445C39.6452 69.0252 39.9187 69.2902 40.2073 69.538C40.7963 70.0929 41.2786 70.7587 41.9062 71.2692C42.6333 71.8592 43.5168 72.2125 44.3451 72.6471C45.1734 73.0817 45.9906 73.6459 46.4066 74.4874C46.5031 74.6592 46.5509 74.8543 46.5447 75.0515Z" fill="#FBBEBE"/>
<path opacity="0.1" d="M52.9962 45.1405C52.9964 46.3082 52.5527 47.4319 51.7557 48.2818C50.9588 49.1317 49.8689 49.6436 48.7088 49.7128C47.5488 49.782 46.4062 49.4033 45.5148 48.6541C44.6234 47.9049 44.0505 46.8418 43.9132 45.6823C43.7759 44.5227 44.0846 43.3543 44.7762 42.4159C45.4678 41.4775 46.4901 40.84 47.6339 40.6338C48.7778 40.4275 49.9568 40.6682 50.9299 41.3065C51.903 41.9448 52.5967 42.9325 52.8692 44.0677C52.9537 44.419 52.9963 44.7791 52.9962 45.1405Z" fill="black"/>
<path d="M52.9962 44.9554C52.9964 46.1231 52.5527 47.2468 51.7557 48.0967C50.9588 48.9466 49.8689 49.4585 48.7088 49.5277C47.5488 49.5969 46.4062 49.2182 45.5148 48.469C44.6234 47.7198 44.0505 46.6568 43.9132 45.4972C43.7759 44.3377 44.0846 43.1692 44.7762 42.2308C45.4678 41.2924 46.4901 40.6549 47.6339 40.4487C48.7778 40.2425 49.9568 40.4831 50.9299 41.1214C51.903 41.7597 52.5967 42.7475 52.8692 43.8827C52.9537 44.2339 52.9963 44.594 52.9962 44.9554Z" fill="#FBBEBE"/>
<path opacity="0.1" d="M38.7311 67.607C38.5213 67.7421 38.2102 67.6977 37.9507 67.6385C36.9272 67.4091 35.7971 67.2445 34.8639 67.6847C34.8624 67.5188 34.9029 67.3552 34.9817 67.2094C35.116 66.9837 35.4032 66.9375 35.6388 66.8746C36.4296 66.6692 37.2504 66.6066 38.0629 66.6897C38.1343 66.6862 38.2049 66.7062 38.264 66.7466C38.3231 66.7869 38.3675 66.8455 38.3906 66.9135C38.5139 67.1336 38.6207 67.3703 38.7311 67.607Z" fill="black"/>
<path d="M36.8076 61.3796C36.5812 62.1194 36.3953 62.8796 35.9517 63.5066C35.8051 63.6715 35.6947 63.8656 35.6278 64.0762C35.6227 64.2318 35.5974 64.386 35.5523 64.5349C35.4897 64.6607 35.3572 64.7439 35.3038 64.8734C35.2247 65.0584 35.3277 65.297 35.2228 65.4708C35.1737 65.5406 35.1148 65.6029 35.048 65.6558C34.7976 65.8888 34.7351 66.2606 34.7295 66.6046C34.724 66.9486 34.7571 67.3 34.6449 67.6237C35.6167 67.0411 36.8537 67.2149 37.958 67.4628C38.2654 67.5312 38.6465 67.583 38.8452 67.3352C39.044 67.0873 38.9299 66.7618 38.9575 66.464C38.9943 66.0756 39.2594 65.7538 39.4232 65.4005C39.6073 65.0029 39.6699 64.559 39.8116 64.1447C40.1043 63.2735 40.7319 62.5633 41.135 61.7402C41.2842 61.4572 41.3733 61.146 41.3964 60.8266C41.3825 60.4125 41.2749 60.007 41.0816 59.641C40.6986 58.7886 40.2429 57.9712 39.7196 57.1977C39.4858 56.8519 39.0827 56.2119 38.7201 56.7649C38.4256 57.2107 38.2415 57.7785 38.0243 58.2668C37.5563 59.2786 37.1499 60.3181 36.8076 61.3796Z" fill="#464353"/>
<path d="M53.4159 54.1996C52.8632 54.2379 52.309 54.1516 51.7938 53.9471C51.2785 53.7427 50.8152 53.4251 50.4377 53.0177C50.3741 52.9596 50.3282 52.8846 50.3052 52.8013C50.2946 52.6931 50.3249 52.5849 50.3898 52.498C50.6107 52.1558 50.9917 51.9302 51.1684 51.5621C51.3451 51.1941 51.2844 50.7705 51.301 50.3673C51.3304 49.6441 51.6139 48.9283 51.6378 48.2163C51.6463 48.034 51.6327 47.8514 51.5973 47.6725C51.5126 47.4068 51.4607 47.1316 51.4427 46.8531C51.4666 46.3112 51.9526 45.8895 52.0335 45.355C52.1385 44.6744 51.5771 44.0825 51.012 43.6904C50.5811 43.3901 50.1133 43.147 49.6204 42.9672C49.3177 42.8582 49.0042 42.782 48.6854 42.7397C48.091 42.659 47.4859 42.727 46.9239 42.9376C46.6913 43.0258 46.471 43.1439 46.2686 43.289C46.1481 43.3656 46.0477 43.4703 45.9759 43.5942C45.9169 43.7337 45.8905 43.8849 45.8986 44.0363C45.8876 44.7927 45.9723 45.5973 45.6133 46.2557C45.4293 46.609 45.1164 46.8957 44.9636 47.2693C44.9011 47.4247 44.862 47.5887 44.8476 47.7557C44.8292 47.929 44.82 48.1031 44.82 48.2773C44.809 49.252 44.82 50.2804 45.3133 51.1256C45.4874 51.3812 45.6454 51.6474 45.7864 51.9228C45.8547 52.0621 45.8864 52.2167 45.8783 52.3718C45.8703 52.5269 45.8228 52.6773 45.7403 52.8087C45.6407 52.9286 45.5147 53.0236 45.3722 53.0862C45.0068 53.2624 44.609 53.3609 44.2039 53.3755C43.7988 53.3902 43.3951 53.3206 43.018 53.1712C43.459 52.7513 43.7619 52.2061 43.8863 51.6086C44.0106 51.011 43.9504 50.3895 43.7138 49.8272C43.5808 49.6001 43.4944 49.3486 43.4598 49.0874C43.4782 48.8521 43.5407 48.6225 43.6438 48.4105C43.8961 47.7155 43.9352 46.9604 43.7561 46.2428C43.6438 45.8026 43.4487 45.3772 43.4137 44.9241C43.3845 44.582 43.3994 44.2376 43.4579 43.8994C43.5588 43.2943 43.8241 42.7289 44.2246 42.2658C44.6251 41.8026 45.1452 41.4597 45.7275 41.2749C45.8213 41.2434 45.9115 41.2138 46.0146 41.1879C46.4612 41.0883 46.9174 41.0393 47.3748 41.0418C48.3964 41.0085 49.4327 40.9789 50.4174 41.2471C50.9956 41.3992 51.5335 41.677 51.993 42.0609C52.3193 42.3365 52.5797 42.6823 52.7551 43.0726C52.8509 43.2975 52.9231 43.5318 52.9704 43.7718C53.0128 43.9697 53.0441 44.1694 53.0754 44.3692C53.2981 45.7915 53.5208 47.2415 53.2833 48.662C53.1895 49.2169 53.0257 49.7717 52.952 50.3377C52.7735 51.6417 53.0956 52.9382 53.4159 54.1996Z" fill="#464353"/>
<path d="M46.876 56.0269C47.0651 56.0269 47.2184 55.8729 47.2184 55.6829C47.2184 55.4929 47.0651 55.3389 46.876 55.3389C46.6869 55.3389 46.5337 55.4929 46.5337 55.6829C46.5337 55.8729 46.6869 56.0269 46.876 56.0269Z" fill="white"/>
<path d="M52.6538 55.8642C52.8429 55.8642 52.9962 55.7102 52.9962 55.5202C52.9962 55.3302 52.8429 55.1761 52.6538 55.1761C52.4648 55.1761 52.3115 55.3302 52.3115 55.5202C52.3115 55.7102 52.4648 55.8642 52.6538 55.8642Z" fill="white"/>
<path opacity="0.1" d="M46.876 38.297L46.1398 36.9006C46.1864 36.8448 46.2375 36.7928 46.2925 36.7452C46.3989 36.6794 46.5234 36.6496 46.6478 36.6602L46.876 38.297Z" fill="black"/>
<path opacity="0.1" d="M48.9063 36.5954L48.0725 37.8346L47.5976 36.4751C47.8002 36.3877 48.0195 36.346 48.24 36.3531C48.4739 36.396 48.6991 36.4779 48.9063 36.5954Z" fill="black"/>
<g opacity="0.1">
<path opacity="0.1" d="M45.7403 52.4425C45.6407 52.5624 45.5147 52.6574 45.3722 52.72C44.7195 53.0338 43.9749 53.0963 43.2794 52.8957C43.1985 52.9945 43.1112 53.0878 43.018 53.1749C43.3951 53.3243 43.7988 53.3939 44.2039 53.3792C44.609 53.3646 45.0068 53.2661 45.3722 53.0899C45.5147 53.0273 45.6407 52.9323 45.7403 52.8124C45.8037 52.7201 45.8469 52.6154 45.8672 52.5051C45.8875 52.3948 45.8844 52.2815 45.8581 52.1725C45.8398 52.2703 45.7995 52.3626 45.7403 52.4425Z" fill="black"/>
<path opacity="0.1" d="M43.7432 46.2484C43.8033 46.4907 43.8385 46.7386 43.8481 46.9882C43.8694 46.6151 43.834 46.2409 43.7432 45.8785C43.6309 45.4383 43.4358 45.0129 43.4009 44.5597C43.4009 44.5024 43.4009 44.4451 43.3917 44.3859C43.3828 44.5671 43.3859 44.7488 43.4009 44.9296C43.4358 45.3828 43.6309 45.8082 43.7432 46.2484Z" fill="black"/>
<path opacity="0.1" d="M53.2705 48.6675C53.3486 48.1784 53.3819 47.6831 53.3699 47.1879C53.3634 47.5597 53.3302 47.9306 53.2705 48.2976C53.1766 48.8525 53.0128 49.4073 52.9391 49.9733C52.9012 50.2769 52.887 50.583 52.8968 50.8888C52.9018 50.7064 52.916 50.5243 52.9391 50.3432C53.0128 49.7772 53.1766 49.2279 53.2705 48.6675Z" fill="black"/>
<path opacity="0.1" d="M43.7064 49.461C43.61 49.2789 43.5323 49.0874 43.4745 48.8895C43.4589 48.9562 43.4515 49.0245 43.4524 49.093C43.4871 49.3541 43.5734 49.6057 43.7064 49.8328C43.8416 50.1429 43.9195 50.475 43.9365 50.8131C43.9698 50.3503 43.8909 49.8864 43.7064 49.461Z" fill="black"/>
<path opacity="0.1" d="M51.5973 47.6651C51.6135 47.7494 51.6251 47.8346 51.6323 47.9203C51.6323 47.8925 51.6323 47.8666 51.6323 47.8389C51.6408 47.6567 51.6272 47.4741 51.5918 47.2951C51.5513 47.1009 51.485 46.9086 51.4537 46.7125C51.4446 46.7564 51.4391 46.8009 51.4372 46.8457C51.4569 47.1244 51.5107 47.3996 51.5973 47.6651Z" fill="black"/>
<path opacity="0.1" d="M50.4377 52.6404C50.4094 52.6106 50.3829 52.5791 50.3586 52.546C50.3133 52.6201 50.2945 52.7076 50.3052 52.7939C50.3282 52.8771 50.3741 52.9521 50.4377 53.0103C50.8152 53.4177 51.2785 53.7352 51.7938 53.9397C52.309 54.1441 52.8632 54.2304 53.4159 54.1921C53.3846 54.0701 53.3533 53.9498 53.3239 53.8222C52.7863 53.8471 52.2498 53.7542 51.7515 53.5502C51.2531 53.3461 50.8049 53.0357 50.4377 52.6404Z" fill="black"/>
<path opacity="0.1" d="M52.0299 45.0036C52.029 44.9968 52.029 44.99 52.0299 44.9832C52.1348 44.3026 51.5734 43.7107 51.0083 43.3186C50.5774 43.0183 50.1097 42.7753 49.6168 42.5954C49.314 42.4865 49.0006 42.4102 48.6817 42.3679C48.0873 42.2872 47.4822 42.3552 46.9202 42.5658C46.6876 42.654 46.4673 42.7722 46.2649 42.9173C46.1444 42.9939 46.044 43.0985 45.9723 43.2224C45.9133 43.3619 45.8868 43.5131 45.895 43.6645C45.895 43.7921 45.895 43.9197 45.895 44.0492V44.0344C45.8868 43.883 45.9133 43.7318 45.9723 43.5923C46.044 43.4685 46.1444 43.3638 46.2649 43.2872C46.4673 43.1421 46.6876 43.0239 46.9202 42.9357C47.4822 42.7251 48.0873 42.6571 48.6817 42.7378C49.0006 42.7801 49.314 42.8564 49.6168 42.9653C50.1097 43.1452 50.5774 43.3882 51.0083 43.6885C51.4777 44.0066 51.9397 44.4653 52.0299 45.0036Z" fill="black"/>
<path opacity="0.1" d="M45.6133 45.8895C45.4292 46.2427 45.1163 46.5294 44.9635 46.903C44.901 47.0585 44.8619 47.2224 44.8476 47.3895C44.8292 47.5627 44.8199 47.7368 44.82 47.911C44.82 48.0683 44.82 48.2273 44.82 48.3864C44.82 48.3512 44.82 48.3161 44.82 48.281C44.8199 48.1067 44.8292 47.9326 44.8476 47.7594C44.8619 47.5923 44.901 47.4284 44.9635 47.2729C45.1163 46.903 45.4219 46.6126 45.6133 46.2594C45.9096 45.7045 45.9041 45.0664 45.8986 44.432C45.923 44.9339 45.8251 45.4342 45.6133 45.8895Z" fill="black"/>
</g>
<path opacity="0.1" d="M52.9575 43.7847C52.9322 43.8209 52.9025 43.8538 52.8692 43.8827C52.8042 43.9364 52.7316 43.98 52.6538 44.0122C52.1182 44.2452 52.1403 43.485 51.8605 43.2242C51.6811 43.0913 51.4609 43.0258 51.2383 43.0393C50.6996 43.0122 50.1603 42.9881 49.6204 42.9672C48.7203 42.9339 47.8221 42.9172 46.9238 42.9376C46.4931 42.945 46.0636 42.9628 45.6354 42.9912C45.1384 43.0245 44.6046 43.0892 44.1813 43.326C44.0363 43.4061 43.9052 43.5091 43.7929 43.6311C43.6917 43.7421 43.5849 43.8827 43.445 43.9086C43.5459 43.3035 43.8112 42.7381 44.2117 42.275C44.6122 41.8118 45.1323 41.4689 45.7145 41.2841C46.6571 40.5786 47.8346 40.2664 49.0009 40.4129C50.1672 40.5594 51.232 41.1532 51.9728 42.0701C52.299 42.3457 52.5595 42.6915 52.7348 43.0818C52.8334 43.3076 52.908 43.5432 52.9575 43.7847Z" fill="black"/>
<path d="M52.6538 43.6422C52.1182 43.8753 52.1403 43.1151 51.8605 42.8543C51.6811 42.7213 51.4609 42.6559 51.2384 42.6694C49.3732 42.5461 47.5024 42.53 45.6354 42.6213C44.9709 42.6657 44.242 42.7655 43.7948 43.2612C43.6806 43.387 43.561 43.5498 43.3898 43.5442C43.3327 43.1743 43.2923 42.7896 43.2646 42.4104C43.248 42.2684 43.2554 42.1245 43.2867 41.985C43.3327 41.826 43.4377 41.6909 43.491 41.5356C43.5436 41.338 43.5623 41.1328 43.5463 40.9289C43.5543 40.1743 43.7561 39.4346 44.1321 38.7814C44.5081 38.1282 45.0456 37.5835 45.6925 37.2002C45.8548 37.1204 46.0054 37.0183 46.1397 36.8969C46.1864 36.841 46.2375 36.7891 46.2925 36.7415C46.3988 36.6757 46.5233 36.6459 46.6478 36.6564C46.7361 36.6564 46.8318 36.6564 46.9091 36.6564C47.1472 36.6301 47.3799 36.5677 47.5994 36.4715C47.802 36.384 48.0213 36.3423 48.2418 36.3494C48.4758 36.3923 48.701 36.4742 48.9081 36.5917C49.0077 36.6399 49.1119 36.6777 49.2192 36.7045C49.3867 36.7434 49.5597 36.7415 49.7272 36.7748C49.9985 36.8442 50.2562 36.9593 50.4892 37.1151C50.7674 37.2659 51.0248 37.4524 51.2549 37.67C51.5052 37.9574 51.7055 38.285 51.8476 38.6392C52.0914 39.1284 52.2768 39.645 52.3998 40.178C52.5489 40.9437 52.4753 41.7723 52.8287 42.4678C53.0441 42.9043 53.241 43.3888 52.6538 43.6422Z" fill="#FED253"/>
<path opacity="0.1" d="M43.3235 43.0615C43.3235 43.0615 47.1392 39.5732 52.8213 42.4659C52.8213 42.4659 46.4066 40.9992 43.3235 43.0615Z" fill="black"/>
<path d="M1.7468 60.8857C1.22773 60.0312 0.447291 59.2488 0.568775 58.2797C1.99712 58.4443 3.35185 58.0947 4.66056 57.4862C5.96926 56.8777 7.1841 56.0879 8.40998 55.313C8.89223 55.0078 9.37264 54.7082 9.85121 54.4307C10.6096 53.9942 11.3624 53.5059 12.1336 53.0787C13.238 52.4683 14.3645 51.969 15.6051 51.8728C15.8298 51.8552 16.0553 51.8496 16.2806 51.8561C17.8084 51.8857 19.3913 52.2982 20.735 51.6101L20.8178 51.5658C21.5799 51.1459 22.1339 50.432 22.7873 49.8531C23.2571 49.4448 23.7686 49.0877 24.3133 48.7877C25.3054 48.2328 26.3803 47.7889 27.3945 47.2655C27.6541 47.1312 27.9006 46.9726 28.1308 46.792L28.3038 46.6533L28.3425 46.62C28.3848 46.5849 28.4271 46.5497 28.4676 46.5127L28.5468 46.4425L28.6278 46.3685L28.6903 46.3093L28.7916 46.2131L28.8026 46.2002L28.9628 46.0411L29.0272 45.9764L29.1395 45.8599L29.2057 45.7933C29.318 45.6767 29.4303 45.5584 29.5444 45.4437C29.6585 45.329 29.8131 45.1737 29.9512 45.046L30.0837 44.9277L30.1224 44.8944L30.2291 44.8056L30.2788 44.7631L30.3837 44.6817L30.4353 44.641C30.4868 44.6022 30.5402 44.5652 30.5936 44.5282C30.6427 44.494 30.6944 44.4637 30.7482 44.4375L30.404 43.5997L31.3943 43.1891L45.0519 76.4904L44.0598 76.901L36.2481 57.8506L36.1929 57.9061C34.3833 59.7004 32.1666 61.0257 29.734 61.7679H29.7193C28.359 62.1804 26.8368 62.4726 25.6919 63.2383C25.2859 63.5035 24.9332 63.8429 24.6519 64.2389C24.3832 64.6237 24.1955 65.062 23.9157 65.4319C23.8623 65.5004 23.8034 65.5669 23.7445 65.6298C23.4097 65.9633 23.0394 66.2585 22.6401 66.5102L22.4652 66.6304C21.2793 67.4689 20.0438 68.2342 18.7655 68.922C18.0279 69.3079 17.2587 69.6296 16.4665 69.8838C15.8904 70.0688 15.2959 70.2038 14.7197 70.3869C14.3322 70.5048 13.9558 70.6571 13.5951 70.8419C12.6196 71.3542 11.8704 72.1569 11.1618 73.0133C10.5415 73.7642 9.95245 74.5595 9.2714 75.2438C8.84437 75.6692 8.19462 76.0724 7.67923 75.8228L7.57432 75.7617C7.59088 75.734 7.60377 75.7062 7.61849 75.6766C8.02528 74.8203 7.52462 73.7623 6.85094 73.0762C6.78099 73.004 6.70921 72.9337 6.63558 72.8672C6.56195 72.8006 6.42943 72.6822 6.32451 72.5897C5.74838 72.0922 5.13545 71.6298 4.73602 70.988C4.43116 70.4489 4.22424 69.8595 4.12492 69.2476C3.94086 68.3764 3.81753 67.4794 3.34633 66.7377C3.0371 66.2476 2.57693 65.8296 2.24193 65.3598C2.03017 65.0818 1.89097 64.755 1.83699 64.4091C1.76152 63.808 2.03946 63.2198 2.14438 62.6206C2.17559 62.4573 2.18855 62.2911 2.18303 62.1249C2.14458 61.6813 1.99446 61.2549 1.7468 60.8857Z" fill="url(#paint0_linear)"/>
<path d="M43.7127 76.6153L44.6878 76.2114L31.2577 43.4699L30.2826 43.8738L43.7127 76.6153Z" fill="#D6D8E1"/>
<path d="M3.67577 66.6212C4.13778 67.3518 4.2611 68.2321 4.43964 69.0885C4.53867 69.6905 4.74241 70.2702 5.04154 70.8012C5.43544 71.4319 6.03733 71.8869 6.60242 72.3752C6.70549 72.4658 6.80857 72.5601 6.90797 72.6489C6.97975 72.7136 7.05154 72.7839 7.11964 72.8523C7.78228 73.5274 8.27373 74.5687 7.87431 75.4103C7.86205 75.4389 7.8479 75.4667 7.83197 75.4935C7.86431 75.5156 7.89814 75.5354 7.93321 75.5527C8.44123 75.8005 9.07994 75.4047 9.49961 74.9849C10.1696 74.3116 10.7494 73.5311 11.3587 72.7932C12.0544 71.9498 12.7925 71.16 13.7515 70.6569C14.1055 70.4747 14.4752 70.3249 14.8559 70.2093C15.421 70.0244 16.0063 69.893 16.5733 69.7155C17.3519 69.4652 18.1081 69.1491 18.8336 68.7704C20.0925 68.0933 21.3096 67.3404 22.4781 66.5158L22.6511 66.3974C23.0435 66.1503 23.4077 65.8606 23.7371 65.5336C23.796 65.4708 23.8531 65.406 23.9064 65.3376C24.1899 64.9677 24.3758 64.5441 24.6427 64.165C24.9195 63.7755 25.2667 63.4417 25.6661 63.181C26.7926 62.4282 28.2872 62.1397 29.6254 61.7346H29.6401C32.0313 61.0037 34.2104 59.7007 35.9903 57.9375C36.1379 57.8137 36.2481 57.651 36.3088 57.4677C36.3658 57.2069 36.2259 56.948 36.0916 56.7186C35.9351 56.4486 35.7823 56.1786 35.6314 55.9048C34.9526 54.6634 34.3466 53.3832 33.8165 52.0707C33.7006 51.7822 33.5883 51.4955 33.4797 51.2051C33.2011 50.4653 32.9471 49.715 32.7177 48.9542C32.6422 48.7026 32.5686 48.4492 32.4986 48.1959C32.4857 48.1478 32.471 48.096 32.4581 48.0405C32.1526 46.8068 31.7439 43.9197 30.4849 44.7834C30.1105 45.0524 29.7623 45.3565 29.445 45.6916C29.2002 45.9412 28.9609 46.1983 28.7087 46.4462C28.5033 46.6491 28.2864 46.84 28.059 47.0177C27.8282 47.1953 27.5817 47.3513 27.3227 47.4838C26.3232 47.998 25.2667 48.4289 24.293 48.9801C23.7568 49.2745 23.2533 49.6254 22.791 50.0269C22.1486 50.5966 21.6038 51.2994 20.8546 51.7119L20.7736 51.7563C19.452 52.4314 17.8967 52.0263 16.3929 51.9967C16.1719 51.9911 15.9507 51.9967 15.7302 52.0133C14.5099 52.1058 13.3944 52.5978 12.3158 53.1989C11.5575 53.6206 10.8175 54.0978 10.0721 54.5288C9.60085 54.8006 9.12964 55.0947 8.65475 55.3962C7.45096 56.1582 6.2306 56.9572 4.97343 57.5343C3.68497 58.1224 2.35418 58.4739 0.949758 58.313C0.830116 59.2655 1.59583 60.0349 2.10753 60.8746C2.3533 61.2355 2.50228 61.6541 2.54009 62.0898C2.54685 62.2527 2.53512 62.4159 2.50511 62.5762C2.40204 63.1662 2.13698 63.7433 2.20325 64.337C2.25548 64.6755 2.39156 64.9954 2.59899 65.2673C2.91926 65.7278 3.37022 66.1403 3.67577 66.6212Z" fill="#F86D70"/>
<path d="M41.0982 70.4739C41.0982 70.4739 42.7106 71.8869 44.3451 72.6397C45.9796 73.3925 44.8973 76.0929 44.8973 76.0929C44.8973 76.0929 43.1653 76.4924 41.9523 74.7982C40.7393 73.104 41.0982 70.4739 41.0982 70.4739Z" fill="#FBBEBE"/>
<path d="M127.985 94.5274C127.985 94.5274 123.714 100.174 120.171 100.22C116.628 100.267 114.548 102.599 116.23 104.056C117.912 105.514 136.245 103.224 136.245 103.224C136.245 103.224 136.418 96.2845 134.83 95.2709C133.241 94.2573 127.985 94.5274 127.985 94.5274Z" fill="#444176"/>
<path d="M173.9 101.306C173.9 101.306 174.553 105.778 172.534 108.828C170.515 111.878 176.182 113.054 176.182 113.054C176.182 113.054 180.3 114.207 180.6 110.119C180.9 106.032 180.802 102.906 180.416 102.573C180.029 102.24 179.707 99.941 179.707 99.941L173.9 101.306Z" fill="#444176"/>
<path d="M109.256 109.753C109.256 109.753 109.107 95.2876 123.352 89.4078L126.659 83.0842L128.246 79.2001C128.821 77.7933 129.724 76.5466 130.881 75.5658C132.037 74.585 133.412 73.8991 134.889 73.5663L137.455 72.9874L138.248 69.6286C138.671 67.8332 139.607 66.2015 140.942 64.9348C142.277 63.6681 143.952 62.8218 145.76 62.5004C147.32 62.2094 148.903 62.0608 150.49 62.0565C151.192 62.0671 151.88 62.2601 152.486 62.6166C153.092 62.9731 153.596 63.481 153.949 64.091C153.949 64.091 167.445 63.9412 168.043 71.3247C168.043 71.3247 169.394 78.1033 171.343 79.3092C171.343 79.3092 174.642 78.5564 177.94 83.3782L181.539 95.7333C181.539 95.7333 190.985 104.622 188.135 112.749C188.135 112.749 184.237 118.324 164.444 114.556C164.444 114.556 143.303 121.03 132.049 112.152C132.047 112.163 113.305 115.779 109.256 109.753Z" fill="#A8A8A8"/>
<path opacity="0.1" d="M118.702 107.493C118.702 107.493 121.702 103.274 121.551 100.259C121.401 97.2444 122.258 89.9016 122.258 89.9016" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M129.649 96.1919C129.649 96.1919 132.647 90.6174 132.347 87.6044C132.347 87.6044 135.496 81.8708 137.895 82.3295" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M146.143 80.3708C146.143 80.3708 154.389 77.3579 154.84 74.9479" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M149.141 87.303C149.141 87.303 157.838 83.3856 157.838 82.3295H162.184C162.184 82.3295 168.483 90.6174 167.734 94.2333" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M159.338 93.3288C159.338 93.3288 150.941 92.7259 147.941 95.1377C144.941 97.5495 139.544 98.7536 139.544 98.7536C139.544 98.7536 136.395 105.534 135.194 106.738C133.994 107.942 132.045 112.163 132.045 112.163" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M159.338 101.315C159.338 101.315 160.237 104.478 161.437 106.287C162.637 108.096 164.435 114.573 164.435 114.573" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M173.432 110.506C173.432 110.506 175.336 115.827 177.009 115.879" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M160.965 87.6655C160.965 87.6655 165.909 89.0083 167.977 92.2617C170.044 95.5151 176.357 98.4651 176.357 98.4651" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M163.086 71.6372C163.086 71.6372 163.086 78.5675 165.035 78.5675C166.984 78.5675 171.334 79.3203 171.334 79.3203" stroke="black" stroke-miterlimit="10"/>
<path opacity="0.1" d="M153.533 63.4066C153.533 63.4066 153.034 75.3788 150.485 78.4787" stroke="black" stroke-miterlimit="10"/>
<g opacity="0.05">
<path opacity="0.05" d="M181.229 95.589L177.631 83.234C174.332 78.4122 171.034 79.1649 171.034 79.1649C169.085 77.959 167.735 71.1785 167.735 71.1785C167.135 63.7951 153.64 63.9467 153.64 63.9467C153.286 63.3357 152.781 62.827 152.174 62.4705C151.566 62.1139 150.877 61.9216 150.174 61.9122C149.537 61.9122 148.885 61.9326 148.234 61.9862C148.602 62.0232 148.955 62.0756 149.292 62.1434L150.341 63.9523C150.341 63.9523 163.835 63.8006 164.435 71.1841C164.435 71.1841 165.786 77.9646 167.735 79.1705C167.735 79.1705 171.034 78.4159 174.332 83.2395L177.931 95.5946C177.931 95.5946 187.377 104.484 184.528 112.611C184.528 112.611 182.604 115.361 174.603 115.694C185.391 116.091 187.826 112.611 187.826 112.611C190.676 104.478 181.229 95.589 181.229 95.589Z" fill="black"/>
<path opacity="0.05" d="M128.448 112.011C125.274 112.585 122.065 112.938 118.842 113.067C122.599 113.212 126.507 112.803 129.025 112.455C128.829 112.305 128.638 112.161 128.448 112.011Z" fill="black"/>
<path opacity="0.05" d="M160.837 114.423C156.22 115.724 151.469 116.479 146.678 116.672C153.59 116.924 160.137 115.475 162.836 114.78C162.184 114.671 161.523 114.547 160.837 114.423Z" fill="black"/>
</g>
<path d="M41.0191 137.924C41.0191 137.924 45.144 137.796 46.3864 136.906C47.6289 136.017 52.7293 134.955 53.0386 136.381C53.3478 137.807 59.2361 143.474 54.5792 143.513C49.9223 143.552 43.7617 142.784 42.5211 142.024C41.2804 141.264 41.0191 137.924 41.0191 137.924Z" fill="#A8A8A8"/>
<path opacity="0.2" d="M54.6639 143.016C50.007 143.054 43.8445 142.287 42.6039 141.536C41.6596 140.957 41.2823 138.882 41.1571 137.926H41.0191C41.0191 137.926 41.2804 141.268 42.5211 142.026C43.7617 142.784 49.9242 143.552 54.5792 143.515C55.9247 143.504 56.3886 143.023 56.3628 142.311C56.1769 142.74 55.6634 143.008 54.6639 143.016Z" fill="black"/>
<path d="M141.581 127.529C141.581 127.529 136.724 126.552 135.43 125.25C134.136 123.948 128.307 121.673 127.659 123.299C127.011 124.925 118.917 130.457 124.421 131.437C129.925 132.417 137.372 132.732 138.992 132.088C140.611 131.445 141.581 127.529 141.581 127.529Z" fill="#A8A8A8"/>
<path opacity="0.2" d="M124.421 130.823C129.926 131.798 137.372 132.118 138.992 131.472C140.223 130.977 141.081 128.596 141.42 127.488L141.581 127.524C141.581 127.524 140.61 131.426 138.992 132.077C137.374 132.728 129.926 132.403 124.421 131.426C122.833 131.145 122.378 130.483 122.549 129.647C122.685 130.198 123.239 130.618 124.421 130.823Z" fill="black"/>
<path opacity="0.1" d="M95.0699 21.2422H93.914C94.1288 21.2422 94.3348 21.1565 94.4867 21.0039C94.6386 20.8513 94.7239 20.6443 94.7239 20.4284C94.7239 20.2126 94.6386 20.0056 94.4867 19.853C94.3348 19.7004 94.1288 19.6146 93.914 19.6146H84.5487C84.3339 19.6146 84.1279 19.7004 83.9761 19.853C83.8242 20.0056 83.7389 20.2126 83.7389 20.4284C83.7389 20.6443 83.8242 20.8513 83.9761 21.0039C84.1279 21.1565 84.3339 21.2422 84.5487 21.2422H85.7047C85.4968 21.2522 85.3007 21.3422 85.1571 21.4936C85.0135 21.6449 84.9334 21.846 84.9334 22.0551C84.9334 22.2642 85.0135 22.4653 85.1571 22.6167C85.3007 22.768 85.4968 22.858 85.7047 22.868H84.0867C83.872 22.868 83.666 22.9537 83.5141 23.1064C83.3622 23.259 83.2769 23.466 83.2769 23.6818C83.2769 23.8977 83.3622 24.1046 83.5141 24.2573C83.666 24.4099 83.872 24.4956 84.0867 24.4956H93.452C93.6668 24.4956 93.8728 24.4099 94.0247 24.2573C94.1766 24.1046 94.2619 23.8977 94.2619 23.6818C94.2619 23.466 94.1766 23.259 94.0247 23.1064C93.8728 22.9537 93.6668 22.868 93.452 22.868H95.0699C95.1794 22.8733 95.2888 22.8561 95.3915 22.8176C95.4942 22.7792 95.5881 22.7201 95.6674 22.6441C95.7467 22.5681 95.8099 22.4767 95.853 22.3755C95.8962 22.2743 95.9184 22.1653 95.9184 22.0551C95.9184 21.945 95.8962 21.836 95.853 21.7348C95.8099 21.6335 95.7467 21.5421 95.6674 21.4661C95.5881 21.3901 95.4942 21.3311 95.3915 21.2926C95.2888 21.2541 95.1794 21.237 95.0699 21.2422Z" fill="#2F80ED"/>
<path opacity="0.1" d="M131.445 49.8752H130.289C130.497 49.8653 130.693 49.7753 130.837 49.6239C130.98 49.4726 131.06 49.2715 131.06 49.0623C131.06 48.8532 130.98 48.6521 130.837 48.5008C130.693 48.3494 130.497 48.2594 130.289 48.2495H120.915C120.805 48.2442 120.696 48.2613 120.593 48.2998C120.49 48.3383 120.396 48.3973 120.317 48.4733C120.238 48.5493 120.175 48.6407 120.132 48.742C120.088 48.8432 120.066 48.9522 120.066 49.0623C120.066 49.1725 120.088 49.2815 120.132 49.3827C120.175 49.484 120.238 49.5753 120.317 49.6513C120.396 49.7273 120.49 49.7864 120.593 49.8249C120.696 49.8633 120.805 49.8805 120.915 49.8752H122.071C121.863 49.8852 121.667 49.9752 121.523 50.1265C121.379 50.2779 121.299 50.479 121.299 50.6881C121.299 50.8972 121.379 51.0983 121.523 51.2497C121.667 51.401 121.863 51.491 122.071 51.501H120.453C120.238 51.501 120.032 51.5867 119.88 51.7393C119.728 51.892 119.643 52.099 119.643 52.3148C119.643 52.5306 119.728 52.7376 119.88 52.8902C120.032 53.0429 120.238 53.1286 120.453 53.1286H129.816C130.031 53.1286 130.237 53.0429 130.389 52.8902C130.541 52.7376 130.626 52.5306 130.626 52.3148C130.626 52.099 130.541 51.892 130.389 51.7393C130.237 51.5867 130.031 51.501 129.816 51.501H131.445C131.554 51.5062 131.664 51.4891 131.767 51.4506C131.869 51.4121 131.963 51.3531 132.042 51.2771C132.122 51.2011 132.185 51.1097 132.228 51.0085C132.271 50.9072 132.293 50.7982 132.293 50.6881C132.293 50.578 132.271 50.469 132.228 50.3677C132.185 50.2665 132.122 50.1751 132.042 50.0991C131.963 50.0231 131.869 49.9641 131.767 49.9256C131.664 49.8871 131.554 49.87 131.445 49.8752Z" fill="#2F80ED"/>
<path opacity="0.1" d="M77.5451 44.5781H76.3891C76.4986 44.5834 76.608 44.5662 76.7107 44.5277C76.8134 44.4892 76.9073 44.4302 76.9866 44.3542C77.0659 44.2782 77.1291 44.1868 77.1722 44.0856C77.2154 43.9843 77.2376 43.8754 77.2376 43.7652C77.2376 43.6551 77.2154 43.5461 77.1722 43.4448C77.1291 43.3436 77.0659 43.2522 76.9866 43.1762C76.9073 43.1002 76.8134 43.0412 76.7107 43.0027C76.608 42.9642 76.4986 42.9471 76.3891 42.9523H67.022C66.9125 42.9471 66.8031 42.9642 66.7004 43.0027C66.5977 43.0412 66.5039 43.1002 66.4246 43.1762C66.3452 43.2522 66.2821 43.3436 66.2389 43.4448C66.1958 43.5461 66.1735 43.6551 66.1735 43.7652C66.1735 43.8754 66.1958 43.9843 66.2389 44.0856C66.2821 44.1868 66.3452 44.2782 66.4246 44.3542C66.5039 44.4302 66.5977 44.4892 66.7004 44.5277C66.8031 44.5662 66.9125 44.5834 67.022 44.5781H68.178C68.0685 44.5729 67.9591 44.59 67.8564 44.6285C67.7537 44.6669 67.6598 44.726 67.5805 44.802C67.5012 44.878 67.438 44.9694 67.3949 45.0706C67.3517 45.1719 67.3295 45.2808 67.3295 45.391C67.3295 45.5011 67.3517 45.6101 67.3949 45.7114C67.438 45.8126 67.5012 45.904 67.5805 45.98C67.6598 46.056 67.7537 46.115 67.8564 46.1535C67.9591 46.192 68.0685 46.2091 68.178 46.2039H66.5582C66.3434 46.2039 66.1374 46.2896 65.9855 46.4422C65.8336 46.5948 65.7483 46.8018 65.7483 47.0177C65.7483 47.2335 65.8336 47.4405 65.9855 47.5931C66.1374 47.7457 66.3434 47.8315 66.5582 47.8315H75.9234C76.1382 47.8315 76.3442 47.7457 76.4961 47.5931C76.648 47.4405 76.7333 47.2335 76.7333 47.0177C76.7333 46.8018 76.648 46.5948 76.4961 46.4422C76.3442 46.2896 76.1382 46.2039 75.9234 46.2039H77.5414C77.6509 46.2091 77.7603 46.192 77.863 46.1535C77.9657 46.115 78.0595 46.056 78.1389 45.98C78.2182 45.904 78.2813 45.8126 78.3245 45.7114C78.3676 45.6101 78.3899 45.5011 78.3899 45.391C78.3899 45.2808 78.3676 45.1719 78.3245 45.0706C78.2813 44.9694 78.2182 44.878 78.1389 44.802C78.0595 44.726 77.9657 44.6669 77.863 44.6285C77.7603 44.59 77.6509 44.5729 77.5414 44.5781H77.5451Z" fill="#2F80ED"/>
<defs>
<linearGradient id="paint0_linear" x1="-133495" y1="12121.5" x2="-144255" y2="12121.5" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
</defs>
</svg>
`

Reply:
Good idea - updated this messaging

Reply:
I'm gonna resolve this convo if that's alright cause this svg text is monstrous lol